src/components/PostCard/PostHeader/PostHeader.js

```diff
@@ -91,9 +92,12 @@ export default class PostHeader extends PureComponent {
 
     const typesWithTimes = ['offer', 'request', 'resource', 'project']
     const canHaveTimes = typesWithTimes.includes(type)
+
+    const { from, to } = formatDatePair(startTime, endTime, true)
+    const startDate = (startTime && from) && formatStartDate(startTime, from)
+    const endDate = (endTime && to) && formatEndDate(endTime, to)
```

Comment:
Here we check the values and then pass in formatted start/end dates with timezone

Reply:
again i think you can remove the formatStartDate and formatEndDate functions as they are no longer really doing anything